tools/confix/upgrade.go

```diff
@@ -48,8 +48,11 @@ func Upgrade(ctx context.Context, plan transform.Plan, configPath, outputPath st
 		return fmt.Errorf("formatting config: %w", err)
 	}
 
+	// ignore validation for serverv2 by checking any default field found in doc
+	entry := doc.First(strings.Split("store.options.ss-pruning-option", ".")...)
```

Comment:
nit: `isServerV2 := doc.First(strings.Split("store.options.ss-pruning-option", ".")...) == nil` makes it more readable.